include/oneapi/dpl/internal/exclusive_scan_by_segment_impl.h

```diff
@@ -145,7 +141,7 @@ exclusive_scan_by_segment_impl(Policy&& policy, InputIterator1 first1, InputIter
     replace_copy_if(policy1, first2, last2 - 1, _flags.get() + 1, _temp.get() + 1, ::std::negate<FlagType>(), init);
```

Comment:
move `policy1`?

Reply:
Done.

Reply:
This is the fault of the `#if 1`, but I think you should move the one in the  `#else` as well.

Reply:
Done.